src/app/unit-host/unit-host.component.ts

```diff
@@ -444,6 +444,16 @@ export class UnitHostComponent implements OnInit, OnDestroy {
     }, '*');
   }
 
+  sendVopNavigationDeniedNotification(event) {
```

Comment:
In dieser Zeile gibt es bei mir eine Linter-Warnung. Kommt die bei dir auch? Falls nicht, sollten wir mal schauen, dass der bei dir richtig eingerichtet ist. Falls doch, hast du sie ignoriert. Dann bitte mit Begründung.

Reply:
Hier könnte man noch ein Typ für den Paramater dranschreiben.